### PR TITLE
WIP avr: add support for the Digispark Pro

### PR DESCRIPTION
**Pull request by aykevl**
This doesn't work yet and I'm not sure why. @deadprogram maybe you can take a look and see what's wrong? The original Digispark works fine with this PR.

### CODE REVIEW

File: targets/digispark-pro.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+	"inherits": ["avr"],
+	"llvm-target": "avr-atmel-none",
+	"cpu": "attiny167",
+	"build-tags": ["digispark_pro", "attiny167", "attiny"],
+	"cflags": [
+		"-mmcu=attiny167"
+	],
+	"ldflags": [
+		"-mmcu=avr35",
```
Comment (deadprogram) [inline targets/digispark-pro.json]: Should this also be `"-mmcu=attiny167"` like the `cflags` argument above?

Comment (deadprogram) [reply targets/digispark-pro.json]: Just saw https://github.com/tinygo-org/tinygo/pull/946/commits/2381cf9b661bcfc8187332216b2c88af53ba76b7 so probably I am incorrect? or maybe this should be `"-mmcu=avr25"`?

**Comment by deadprogram** [comment]
Played around with it, but was not able to get anything to happen, nor to notice any specific problems. Very odd.

**Comment by deadprogram** [comment]
CCing @ekettenburg just in case he might take an interest in what we are trying to do here.

File: targets/digispark-pro.json (hunk 1 above)
Comment (aykevl) [reply targets/digispark-pro.json]: Exactly, apparently things go wrong when you specify the exact device. You can get the value of `-mmcu=avr*` from here: https://gcc.gnu.org/onlinedocs/gcc/AVR-Options.html (it lists the attiny167 under avr35).

**Comment by aykevl** [comment]
I don't currently have plans to continue working on this PR, so closing. If I want to do it, I can always restore the code from this PR.

**Pull request closed without merging**